composer.json

```diff
@@ -17,6 +17,7 @@
     ],
     "require": {
         "php": "^7.3|^8.0",
+        "alcohol/iso4217": "^4.0",
```

Comment:
Remove this dependency (maybe add it to the dev-deps), and mention in the docs that this dependency should be installed for using the `Currency` rule. 

When the `Currency` rule is used without the package installed, a exception with a nice description should be thrown.

Reply:
@freekmurze What about changing this package to the already installed thephpleague/iso3166 package as mentioned in the PR description? :) I think that package will work for the majority of the usecases and won't require any new packages to be required.

> Note that an alternative could be to rely on the already required package [thephpleague/iso3166](https://github.com/thephpleague/iso3166) which includes currency codes (see [src/ISO3166.php@L2431](https://github.com/thephpleague/iso3166/blob/b69892c2ad05fcf17f0b4ed4542b535081f36cfa/src/ISO3166.php#L2431)), but I was not able to confirm if this list is in fact complete.

Reply:
@freekmurze Following up with a bit of data comparing the two packages:

```php
$alcohol = data_get((new \Alcohol\ISO4217())->getAll(), '*.alpha3');

$theleague = Arr::sort(array_unique(data_get((new League\ISO3166\ISO3166)->all(), '*.currency.*')));
asort($theleague);
$theleague = array_values($theleague);

// Check what is the difference between the two packages:
dump(
    array_diff($alcohol,$theleague), // ['LVL', 'MRU', 'STN', 'VES'] are values in $alcohol that are NOT in $theleague
    array_diff($theleague, $alcohol) // ['BYR', 'MRO', 'STD', 'VEF'] are values in $theleague that are NOT in $alcohol
);
```

So it looks like no matter which one of the packages we choose, we will get a very small error :)

I suggest simply switching to the already required package [League\ISO3166](https://github.com/thephpleague/iso3166) - let me know if that would be acceptable Freek.

Reply:
> let me know if that would be acceptable

ok 👍